src/System.Reflection.MetadataLoadContext/src/System.Reflection.MetadataLoadContext.csproj

```diff
@@ -0,0 +1,173 @@
+
+         Unfortunately, CoreFxLab does not have support for multiple-TFM builds and at the moment, our immediate prospects need NETFX 4.6 support
+         so we're building the inferior netstandard version.
+    -->
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <PropertyGroup>
+    <ProjectGuid>{DBB18588-DD89-4F41-9852-484FD4726F0F}</ProjectGuid>
+    <AssemblyName>System.Reflection.MetadataLoadContext</AssemblyName>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Description>.NET Core Type Loader</Description>
+    <Configurations>netcoreapp-Unix-Debug;netcoreapp-Unix-Release;netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release</Configurations>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="FromCoreRt\CoreRtBridge.cs" />
+    <Compile Include="FromCoreRt\System\DefaultBinder.CanConvert.cs" />
+    <Compile Include="FromCoreRt\System\DefaultBinder.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\AssemblyNameHelpers.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\BindingFlagSupport\ConstructorPolicies.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\BindingFlagSupport\EventPolicies.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\BindingFlagSupport\FieldPolicies.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\BindingFlagSupport\MemberPolicies.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\BindingFlagSupport\MemberTypeIndex.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\BindingFlagSupport\MethodPolicies.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\BindingFlagSupport\NameFilter.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\BindingFlagSupport\NameFilter.Ecma.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\BindingFlagSupport\NestedTypePolicies.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\BindingFlagSupport\PropertyPolicies.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\BindingFlagSupport\QueriedMemberList.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\BindingFlagSupport\QueryResult.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\BindingFlagSupport\QueryResult.Enumerator.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\BindingFlagSupport\Shared.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\General\Assignability.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\General\ListBuilder.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\TypeInfos\RuntimeTypeInfo.BindingFlags.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\TypeInfos\RuntimeTypeInfo.GetMember.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\Runtime\TypeInfos\RuntimeTypeInfo.TypeComponentsCache.cs" />
+    <Compile Include="FromCoreRt\System\Reflection\SignatureTypeExtensions.cs" />
```

Comment:
Is there value in maintaining this "FromCoreRt" separation?

Reply:
I was wondering the same thing; I left it as-is since I wasn't sure if we had a policy regarding this. In the future in theory that code could be placed in a shared location that is sync'd between repos.

Reply:
Agree that the corert prefix is not very helpful. Number of these files diverged from what is in CoreRT.

Reply:
I removed the FromCoreRt separation